src/components/dashboard/GovernanceSection/GovernanceSection.tsx

```diff
@@ -1,14 +1,25 @@
-import { Typography, Grid, Card, Box } from '@mui/material'
+import { Typography, Grid, Card, Box, Paper, Alert } from '@mui/material'
 import { WidgetBody } from '@/components/dashboard/styled'
 import ExpandMoreIcon from '@mui/icons-material/ExpandMore'
 import Accordion from '@mui/material/Accordion'
 import AccordionSummary from '@mui/material/AccordionSummary'
 import AccordionDetails from '@mui/material/AccordionDetails'
 import css from './styles.module.css'
+import SafeAppsErrorBoundary from '@/components/safe-apps/SafeAppsErrorBoundary'
+import SafeAppsLoadError from '@/components/safe-apps/SafeAppsErrorBoundary/SafeAppsLoadError'
+import AppFrame from '@/components/safe-apps/AppFrame'
+import { useBrowserPermissions } from '@/hooks/safe-apps/permissions'
+import { useRemoteSafeApps } from '@/hooks/safe-apps/useRemoteSafeApps'
+import { SafeAppsTag } from '@/config/constants'
 
-const GovernanceSection = () => (
-  <Grid item xs={12} md>
-    <Accordion className={css.accordion}>
+const GovernanceSection = () => {
+  const { getAllowedFeaturesList } = useBrowserPermissions()
+  const [claimingSafeApp, errorFetchingClaimingSafeApp] = useRemoteSafeApps(SafeAppsTag.SAFE_CLAIMING_APP)
+  const claimingApp = claimingSafeApp?.[0]
+  const fetchingSafeClaimingApp = !claimingApp && !errorFetchingClaimingSafeApp
```

Comment:
I've implemented this loading logic because the loading state returned by `useRemoteSafeApps` starts false.

Reply:
Can we not move this logic inside the hook?

Reply:
Can we not move this logic into the hook? This will forever load if an error occurs.